pkg/test/resourcefixture/testdata/sensitivefield/sensitivevaluesimple/_generated_object_sensitivevaluesimple.golden.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeSSLCertificate
+metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/observed-secret-versions: (removed)
+    cnrm.cloud.google.com/project-id: ${projectId}
+    cnrm.cloud.google.com/state-into-spec: absent
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 2
+  labels:
+    cnrm-test: "true"
+  name: computesslcertificate-${uniqueId}
+  namespace: ${uniqueId}
+spec:
+  certificate:
+    value: |
```

Comment:
Ops. You may want to change this to fake data.

Reply:
This is the test certificate in [create.yaml](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/f6ffa102c61d86b92572951aae7f385e89994569/pkg/test/resourcefixture/testdata/sensitivefield/sensitivevaluesimple/create.yaml#L22)